oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -300,8 +301,44 @@ void shouldRedirectToLoginWhenUserHasNoExistingSession() {
 
             var requestParams = buildRequestParams(null);
             var event = withRequestEvent(requestParams);
+            event.setRequestContext(
+                    new ProxyRequestContext()
+                            .withIdentity(new RequestIdentity().withSourceIp("123.123.123.123")));
+            var response = makeHandlerRequest(event);
+
+            assertThat(response, hasStatus(302));
+            var locationHeader = response.getHeaders().get(ResponseHeaders.LOCATION);
+            verify(orchestrationAuthorizationService)
+                    .storeState(eq(session.getSessionId()), any(State.class));
+            assertThat(locationHeader, containsString(TEST_ENCRYPTED_JWT.serialize()));
+            assertThat(
+                    splitQuery(locationHeader).get("request"),
+                    equalTo(TEST_ENCRYPTED_JWT.serialize()));
+            assertThat(splitQuery(locationHeader).get("client_id"), equalTo(orchClientId));
+            assertThat(
+                    splitQuery(locationHeader).get("response_type"),
+                    equalTo(ResponseType.CODE.toString()));
+            var captor = ArgumentCaptor.forClass(JWTClaimsSet.class);
+            verify(orchestrationAuthorizationService).getSignedAndEncryptedJWT(captor.capture());
+            assertEquals(
+                    "{\"userinfo\":{\"email_verified\":null,\"email\":null}}",
+                    captor.getValue().getClaim("claim"));
+        }
+
+        @Test
+        void shouldPassTheCorrectClaimsToAuth()
+                throws com.nimbusds.oauth2.sdk.ParseException, ParseException {
+            var orchClientId = "orchestration-client-id";
+            when(configService.isAuthOrchSplitEnabled()).thenReturn(true);
+            when(configService.getOrchestrationClientId()).thenReturn(orchClientId);
+            when(orchestrationAuthorizationService.getVtrList(any())).thenCallRealMethod();
             when(orchestrationAuthorizationService.getSignedAndEncryptedJWT(any()))
                     .thenReturn(TEST_ENCRYPTED_JWT);
+
+            var requestParams =
+                    buildRequestParams(
+                            Map.of("scope", "email openid profile phone", "vtr", "[\"Cl.Cm.P2\"]"));
```

Comment:
Should the test exclude email to test that it get requested to auth whether or not it is present?

Reply:
Good shout